zgw/taak/src/main/kotlin/nl/nlportal/zgw/taak/domain/TaakObject.kt

```diff
@@ -13,19 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.ritense.portal.taak.domain
+package nl.nlportal.zgw.taak.domain
 
 import com.fasterxml.jackson.annotation.JsonInclude
 import com.fasterxml.jackson.annotation.JsonProperty
 import java.util.UUID
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
-data class ObjectsApiTaak(
-    val kvk: String?,
-    val bsn: String?,
+data class TaakObject(
+    val identificatie: TaakIdentificatie,
     val data: Map<String, Any>,
     @JsonProperty("verzonden_data") var verzondenData: Map<String, Any>? = null,
     var status: TaakStatus,
-    @JsonProperty("formulier_id") val formulierId: String,
+    val formulier: TaakFormulier,
     @JsonProperty("verwerker_taak_id") val verwerkerTaakId: UUID,
+)
+data class TaakIdentificatie(
```

Comment:
Please move these to a separate file

Reply:
Fixed in previous (and already merged) PR